src/features/admin/pages/AdminStatistics.css

```
@@ -0,0 +1,140 @@
+
+.dropdown-button {
+  border: 1px solid #ccc;
+  padding: 8px 16px;
+  border-radius: 8px;
+  background: #f5f5f5;
+  cursor: pointer;
+  min-width: 140px;
+}
+
+.dropdown-menu {
+  position: absolute;
```

Stylelint (17.5.0)

[error] Unexpected duplicate "position" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)